### DISCUSSION TITLE
How to check Hacktribe version and (safely) update if needed?

### DISCUSSION THREAD

**Discussion by parrucasco** [bangcorrupt/hacktribe#347] [Q&A] [answered]
well, what's in the title...i'm much of a noob when it comes to hacktribe and electribe in general. i bought a used E2 four months ago to inmediately install hacktribe and i was totally inlove with it. 
i don't need to go back to OG firmware, but maybe i will out of curiosity and to record sounds and samples to collect.
i'm also curious to check if there are fixes to some minor bugs that i've read...and why not, check if the long expected "unquantized recording" is added to the features ;)

marry christmas to all and thanks in advance!

**Comment by bangcorrupt** [comment, marked as answer]
I haven't got as far as version numbers yet, but the patch hasn't been updated since last year.  If you have the MFX 'Tap Delay Sync', 'Plate Reverb', 'Big Hall' and 'Tube Drive' then you have the latest version.

The easiest way to patch the firmware is using the [Hacktribe Tutorial](https://github.com/bangcorrupt/hacktribe-tutorial) codespace.

The next update to the patch will be a sysex loader for executing Freetribe apps, hopefully in the next few days.
